packages/x-charts/src/internals/plugins/featurePlugins/useChartClosestPoint/useChartClosestPoint.ts

```diff
@@ -56,9 +34,10 @@ export const useChartClosestPoint: ChartPlugin<UseChartClosestPointSignature> =
   const zoomIsInteracting = useSelector(store, selectorChartZoomIsInteracting);
 
   const { series, seriesOrder } = useSelector(store, selectorChartSeriesProcessed)?.scatter ?? {};
-  const voronoiRef = React.useRef<Record<string, VoronoiSeries>>({});
-  const delauneyRef = React.useRef<Delaunay<any> | undefined>(undefined);
-  const lastFind = React.useRef<number | undefined>(undefined);
+  const flatbushMap = useSelector(
+    store,
+    zoomIsInteracting ? selectorChartSeriesEmptyFlatbushMap : selectorChartSeriesFlatbushMap,
```

Comment:
Why not move the `if (zoomIsInteracting) return EMPTY` logic inside the `selectorChartSeriesFlatbushMap`?